LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2016 CloudCannon
```

Comment:
might want to add Copyright (c) Artifice 2018 above the existing CloudCannon copyright.